client/signup/steps/domains/index.jsx

```diff
@@ -659,7 +659,10 @@ export class RenderDomainsStep extends Component {
 			: [];
 		const cartIsLoading = this.props.shoppingCartManager.isLoading;
 
-		if ( cartIsLoading || this.shouldHideUseYourDomain() ) {
+		if (
+			this.shouldHideUseYourDomain() ||
```

Comment:
I'm still seeing the regression in /start/domain/domain-only. I think the issue is `this.shouldHideUseYourDomain()` is true for the domain-only page, but we don't want to hide the entire side bar when it's true, just the "Use your domain" section.